pkgs/top-level/aliases.nix

```diff
@@ -92,6 +92,7 @@ mapAliases ({
   bird2 = bird; # Added 2022-02-21
   bitwig-studio1 = throw "bitwig-studio1 has been removed, you can upgrade to 'bitwig-studio'"; # Added 2023-01-03
   bitwig-studio2 = throw "bitwig-studio2 has been removed, you can upgrade to 'bitwig-studio'"; # Added 2023-01-03
+  blender-with-packages = args: lib.warn "blender-with-packages is deprecated in favor of blender.withPackages, which behaves similarly to python3.withPackages" (blender.withPackages (_: args.packages)).overrideAttrs (lib.optionalAttrs (args ? name) { pname = "blender-" + args.name; }); # Added 2023-10-30
```

Comment:
Maybe add some line breaks here? This line really feels like a lot.

Reply:
> Don't worry about it. Also, the message should probably refer people to blender.withPackages.
told ya. I'll do something about it

Reply:
Does this work?

https://github.com/NixOS/nixpkgs/blob/521cd5a7d283a5ef655eacba58dd488d980a03d4/pkgs/top-level/aliases.nix#L95-L97

Reply:
Definitely more readable, yeah. Also, I'd probably drop the `python3.withPackages` bit entirely and just provide an example - something like "...deprecated in favor of `blender.withPackages`, e.g. `blender.withPackages(ps: [ ps.honk ])`"

Reply:
> Definitely more readable, yeah. Also, I'd probably drop the `python3.withPackages` bit entirely and just provide an example - something like "...deprecated in favor of `blender.withPackages`, e.g. `blender.withPackages(ps: [ ps.honk ])`"

I just feel iffy about referring to a specific package for a general example

Reply:
I don't really see a problem with that personally. If you really wanted to be fancy about it, you could generate the expression from the passed-in package names, I guess (but I feel like this would be trying way too hard lmao)

Reply:
Ah, I know what to do!

Reply:
https://github.com/NixOS/nixpkgs/blob/fe4c776ae9c4289d848096fba1621b3c5c6d0bde/pkgs/top-level/aliases.nix#L96

How's that?